pyrene/.eslintrc

```diff
@@ -15,5 +15,13 @@
     "react/forbid-prop-types": 0, 
     "no-multiple-empty-lines": 0,
     "@typescript-eslint/unbound-method": 0
-  } 
+  },
+  "overrides": [
```

Comment:
Shouldn't this better be part of the `eslint-config` package?
No strong opinion though.